application/models/meeting_test.go

```diff
@@ -139,6 +139,149 @@ func (ms *ModelSuite) TestMeeting_FindByUUID() {
 	}
 }
 
+// TestMeeting_FindCurrent tests the FindCurrent function of the Meeting model
+func (ms *ModelSuite) TestMeeting_FindCurrent() {
+	t := ms.T()
+
+	meetings := createMeetingFixtures_FindByTime(ms)
+
+	nearFuture := time.Time(meetings[1].EndDate) // Also the start date of meetings[2]
+	farFuture := time.Time(meetings[3].EndDate).Add(domain.DurationDay * 2)
+
+	tests := []struct {
+		name    string
+		want    []string
+		testNow time.Time
```

Comment:
Is it worth the complexity to simulate different "now" dates? I guess I don't understand why the varying meeting fixture dates don't adequately test the date logic.

Reply:
I thought it's nice to be able to see it working for empty results, a result with one date and a result with more than one date.

Reply:
Could it also be done with additional meeting fixture records?